### ISSUE TITLE
Neocron City Zone BSP Issues

### ISSUE THREAD

**Issue by hinch** [NCC-Lykos/neocron-classic#298]
The url at the bottom has a zip of all the major city zones (haven't done all subway stations, outzone yet) just focused on the main city zones.

I've screenshotted where something doesn't look right it could be texture misalignment, light map not being applied, light bleed from behind the wall, sometimes it looks like light is being picked up by an invisible poly, some zones are considerably lighter than others, some textures still pending update others are bugs like wrong texture or stuck objects/actors
There's a few random stray lights that don't appear to have a source at least not to a player so they could probably just be removed to simplify things, a few signs that don't have their green glow light source.  Also the shop counters some seem to be picking up the orange glow others dont, couple of textures for logos without any visible frame or projection and some that don't appear to have transparency applied properly and have black backgrounds.  All in its not as terrible as it could be considering the size but a little polish never hurt :)

 http://angel-photography.biz/neocron1.zip

**Comment by ghost** [comment]
#113

**Comment by NCC-Therenall** [comment]
@hinch Your link only goes to a page that says "Coming soon" :)

**Comment by hinch** [comment]
yeah works now i uploaded it under a different name and forgot to rename it :) its been a long day.

**Comment by NCC-Lykos** [comment]
@hazuk This may be a duplicate, but he laid them all out in one go. I asked for this <3

**Comment by hinch** [comment]
http://angel-photography.biz/ppoz_stations.zip

edit: fixed spelling :) stations not sations :)

**Comment by NCC-Therenall** [comment]
@hinch That's `Coming soon` Again btw 😛 

Also thanks for the reports :)

**Comment by hinch** [comment]
fixed the spelling. this time file was right link was wrong :)

**Comment by hinch** [comment]
http://angel-photography.biz/oz_sec_1_2.zip

Hopefully i've got the filename / link right this time 3rd times a charm so they say.
This is OZ 1 & 2 they're not too bad for object placement but almost all the light textures are fubar throughout both zones and where there's height changes the texture transitions are all kinda wrong in some area's either bleeding into different textures within the same sprite/texture sheet or just bad alignment or in some cases the entirely wrong texture all over.

**Comment by NCC-Lykos** [comment]
Slow down home skillet <3 let me get some of these done lol I'll report back when I need more.

**Comment by hinch** [comment]
figured i'd blitz through them all while i had some chance / time and just dump them all here then you can fix them all in your own time at least it saves you the time of searching for everything yourself or for the broken geometry i'm assuming most of the light bleeds where they shouldn't be are polys joined to faces they shouldn't be due to shitty bsp's from whoever made them originally.
